### PR TITLE
New script cleanup-deployment.sh

### PR DESCRIPTION
**Pull request by brc**
Script to safely delete nova instances and quantum networks from the
jenkins build cluster.

### CODE REVIEW

File: cleanup-deployment.sh [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+quantum net-list  >$tmp_networks    # can't --name here, have to grep these
+quantum subnet-list  >$tmp_subnets  #
+
+# when no instances are found, the output file size is 1 byte ('\n').
+# but check for empty file too.
+if [ $(stat $tmp_instances |grep -w Size |awk '{print $2}') = 1 \
+    -o ! -s $tmp_instances ]
+then
+    echo
+    echo -e "[-] ==> \e[1;41;33mNo instances match /$REGEX/\e[0m"
```
Comment (rpedde) [inline cleanup-deployment.sh:50]: WHY YOU DO THIS?!?!?!?!?!

File: cleanup-deployment.sh [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+REGEX=$1
+STATUS=${2:-ACTIVE}
+
+function cleanup
+{
+    rm -f $tmp_instances
+    rm -f $tmp_networks
+    rm -f $tmp_subnets
+}
+trap cleanup EXIT
```
Comment (rpedde) [inline cleanup-deployment.sh:33]: Nicely done, sir.  Thank you.

**Comment by rpedde** [comment]
lgtm +1

File: cleanup-deployment.sh (hunk 1 above)
Comment (brc) [inline cleanup-deployment.sh:50]: Because, the user!!!1

**Pull request merged**